Spanner/src/Connection/Grpc.php

```diff
@@ -455,6 +457,42 @@ public function createSession(array $args)
         ]);
     }
 
+    /**
+     * @return PromiseInterface[]
+     */
+    public function createSessionsAsync($count, array $args)
```

Comment:
Two notes here:

- Could we please add similar annotations here as from the [`deleteAsync`](https://github.com/googleapis/google-cloud-php/blob/c3aa1ba6d60d8ae78e1b225bd087b4a8b781eb81/Spanner/src/Connection/Grpc.php#L521-L527) call?
- Typically our calls in the connection implementation instantiate a single request, to keep consistency let's just have this create a single promise, so no `$count` argument.

Reply:
Sure! I can have the Connection make one request at a time, and have the Operation be responsible for aggregating the promises.